packages/lib/src/core/helpers/stylesConverter.js

```diff
@@ -62,19 +81,89 @@ const layoutParamsMap = {
-  cropOnlyFill: 'layoutParams_cropOnlyFill', //????????????????
   fixedColumns: 'layoutParams_fixedColumns', //????????????????
 
   scatter: 'layoutParams_scatter',
   scrollDirection: 'layoutParams_scrollDirection',
-  minItemSize: 'layoutParams_minItemSize',
 
   isVertical: 'layoutParams_isVerticalOrientation', // This needs to be refactored to be an enum. but can wait
   columnWidths: 'layoutParams_columnWidths',
 
-  externalInfoHeight: 'layoutParams_externalInfoHeight', //layouter internal
-  externalInfoWidth: 'layoutParams_externalInfoWidth', //layouter internal
-  targetItemSize: 'layoutParams_targetItemSize', //layouter internal
+  externalInfoHeight: 'layoutParams_externalInfoHeight', //layouter API
+  externalInfoWidth: 'layoutParams_externalInfoWidth', //layouter API
+  targetItemSize: 'layoutParams_targetItemSize', //layouter API
+};
+
+const behaviourParams = {
+  //item
+  itemClick: 'behaviourParams_item_clickAction', //possible refactor (join fullscreen, expand into one 'action')
+  //----video
+  videoSpeed: 'behaviourParams_item_video_speed',
+  hidePlay: 'behaviourParams_item_video_enablePlayButton',
+  videoPlay: 'behaviourParams_item_video_playTrigger',
+  videoLoop: 'behaviourParams_item_video_loop',
+  showVideoControls: 'behaviourParams_item_video_enableControls',
+  videoSound: 'behaviourParams_item_video_volume',
+  //----overlay
+  hoveringBehaviour: 'behaviourParams_item_overlay_hoveringBehaviour',
+  overlayAnimation: 'behaviourParams_item_overlay_hoverAnimation',
+  overlayPosition: 'behaviourParams_item_overlay_position',
+  overlaySize: 'behaviourParams_item_overlay_size',
+  overlaySizeType: 'behaviourParams_item_overlay_sizeType',
+  overlayPadding: 'behaviourParams_item_overlay_padding',
+  //----content
+  imageHoverAnimation: 'behaviourParams_item_content_hoverAnimation',
+  imagePlacementAnimation: 'behaviourParams_item_content_placementAnimation',
+  imageLoadingMode: 'behaviourParams_item_content_loader',
+  //----info
+  titlePlacement: 'behaviourParams_item_info_placement',
+
+  //gallery
+  scrollSnap: 'behaviourParams_gallery_enableScrollSnap',
+  isRTL: 'behaviourParams_gallery_layoutDirection', // changes from boolean to an enum (refactor)
+  allowLeanGallery: 'behaviourParams_gallery_enableLeanGallery', //think about removing this!
+  allowContextMenu: 'behaviourParams_gallery_disableContextMenu', //REFACTOR reverse
+  //----vertical
+  scrollAnimation: 'behaviourParams_gallery_vertical_scrollAnimation',
+  //--------loadMore
+  enableInfiniteScroll:
+    'behaviourParams_gallery_vertical_loadMore_showLoadMore',
+  loadMoreAmount: 'behaviourParams_gallery_vertical_loadMore_amount',
+  loadMoreButtonText: 'behaviourParams_gallery_vertical_loadMore_buttonText',
+
+  //----horizontal
+  slideAnimation: 'behaviourParams_gallery_horizontal_slideAnimation',
+  enableScroll: 'behaviourParams_gallery_horizontal_blockScroll', //requires a reversal! (blocks instead of allowing),
```

Comment:
This converter should not include changes that require a refactor IMO - let's talk about it on Sunday